docs/blog/2018-12-15-develop-and-deploy-your-own-react-monorepo-app/index.md

```diff
@@ -0,0 +1,598 @@
+- provider — tells the name of the script to run for deployment, in this case it’s ‘script’
+- skip_cleanup — we don’t want Travis to get rid of any files made during the build. We created the ‘public’ folder when running the build, so we don’t want Travis to get rid of it.
+- app — specifies the name of the application we’re deploying,
+
+We have to do add a little bit of configuration on our side to ensure that Now hosts our sites correctly. The config comes in the form of a `now.json` file, which outlines the options we want Now to use when deploying our site. Go into the directory for our blog packages and create a `now.json` file. We want to let Now know that we’re deploying a static site, the entry directory to the static site, and what alias we want to assign our site. Put this inside your `now.json` file.
+
+```
+{
+  type: "static",
+  alias: "lerna-monorepo-blog",
+  static: {
+    public: "public"
+  },
+  public: true
+}
+```
+
+Note: `public: true` needs to be specified since you’ll most likely have a free Now account. Having a publicly facing site is one of the restrictions of the free tier. The entry file to our static site is also called `public` so don’t conflate the two.
+
+Push these changes to your repo to kick off your build pipeline. If everything was successful then your build will have passed and your site will have deployed. Go to Travis, open up your blog build so you have access to the logs. Scroll right to the bottom to where it says ‘deploying application’ and uncollapse the section. This’ll be the URL that Now deployed your site to. It should look something like `https://buildsjdoe383jd.now.sh`. Copy that url and go back to your command line. Run the following command:
+
+`now alias <your randomly generated URL> <your chosen alias>`
+I chose ‘lerna-monorepo-blog’ as my alias, so that one won’t be available for you to use, so choose your own. When you’ve run the command, there will be feedback saying that the alias was successfully created. If not, then it’s likely the command was written incorrectly or the alias is already in use by someone else. Once you’ve added an alias successfully for the blog, do the same for your shop package.
+
+Even if you push everything forward now the site’s navigation will still be broken. The very last thing we need to do is add our alias URL as an environment variables in our `.env.production` file. In your blog package’s `.env.production` go ahead and add the following:
+
+`BLOG_URL=https://<your chosen alias>.now.sh/`
+
+Do the same for your shop url, commit and push. Congrats!
+
+### Conclusion
+
+You should now be able to access the site and navigate easily between your shop and your blog. Great job! If things didn’t work out quite as planned then don’t sweat it. This tutorial is made of tiny moving parts which make for loads of opportunities for things to go wrong. I’ll try to make myself available to answer any messages left in the comments regarding any issues you come across.
```

Comment:
```suggestion
 You should now be able to access the site and navigate easily between your shop and your blog. 
```

Reply:
why?

Reply:
* No email address or contact info
* Implies the tutorial is buggy
* No comments on this blog